internal/pkg/encryption/keys/nodeid.go

```diff
@@ -7,7 +7,7 @@ package keys
 import (
 	"fmt"
 
-	"github.com/talos-systems/go-smbios/smbios"
+	"github.com/talos-systems/talos/internal/app/machined/pkg/controllers/hardware"
```

Comment:
this should be a linter error - we import `internal/` package from public `pkg/`.

In fact I think we need to move `GetSMBIOSInfo()` function to some other shared package.